integration/delete_compat_test.go

```diff
@@ -61,26 +119,38 @@ func testDeleteCompat(t *testing.T, testCases map[string]deleteCompatTestCase) {
 			// Use per-test setup because deletes modify data set.
 			ctx, targetCollections, compatCollections := setup.SetupCompat(t)
 
-			filter := tc.filter
-			require.NotNil(t, filter)
-
-			opts := options.BulkWrite().SetOrdered(tc.ordered)
+			require.NotNil(t, tc.deletes)
 
 			for i := range targetCollections {
 				targetCollection := targetCollections[i]
 				compatCollection := compatCollections[i]
 				t.Run(targetCollection.Name(), func(t *testing.T) {
 					t.Helper()
 
-					dmm := mongo.NewDeleteManyModel().SetFilter(filter)
-
-					targetRes, targetErr := targetCollection.BulkWrite(ctx, []mongo.WriteModel{dmm}, opts)
-					compatRes, compatErr := compatCollection.BulkWrite(ctx, []mongo.WriteModel{dmm}, opts)
+					var targetRes bson.D
+					targetErr := targetCollection.Database().RunCommand(
```

Comment:
Can we keep using BulkWrite?

Reply:
As far as I can see there's no Model for RunCommand, which is required to provide a couple of queries (We need a couple of them to easily simulate an error in one of them)

Reply:
What specific queries we can't run with BulkWrite?

Reply:
We can't run a couple of queries in one request (at least I didn't find a way to do that).